app/brave_main_delegate.cc

```diff
@@ -243,6 +244,7 @@ bool BraveMainDelegate::BasicStartupComplete(int* exit_code) {
     features::kSmsReceiver.name,
     features::kVideoPlaybackQuality.name,
     features::kTabHoverCards.name,
+    network_time::kNetworkTimeServiceQuerying.name,
```

Comment:
Should also be added to https://github.com/brave/brave-core/blob/master/app/brave_main_delegate_browsertest.cc#L63

Reply:
Sorry for the late reply. I was on holidays until yesterday. ok, I add it here. Thanks.